src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1208,4 +1223,30 @@ export class NetworkAreaDiagramViewer {
             }
         }
     }
+
+    private updataMetadata(mousePosition: Point) {
+        // get move node in metadata
+        const node: SVGGraphicsElement | null = this.container.querySelector(
+            'nad\\:node[svgid="' + this.selectedElement?.id + '"]'
+        );
+        if (node != null) {
+            // save original position, for the callback
+            const xOrig = node.getAttribute('x') ?? 0;
+            const yOrig = node.getAttribute('y') ?? 0;
+            // update node position in metadata
+            node.setAttribute('x', mousePosition.x.toFixed(2));
```

Comment:
Here we do this rounding twice instead of introducing a variable. But I realize that the mouse position is rounded to 2 decimals most of the time afterwards (sometimes it's not, but the it's the translation which is rounded). Couldn't we avoid this by simply returning directly this rounded values in `getMousePosition(event)`?

Reply:
Actually the toFixed method does not round the number, it formats the number as string. It is used when setting the data to the svg nodes attributes.  I added a method for formatting these values (and also other 2 methods for points and polyline) to make thing more clear